boards/arm/adafruit_kb2040/adafruit_kb2040-pinctrl.dtsi

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2021, Yonatan Schachter
```

Comment:
Not sure about the rules here, but you should probably add yourself below or replace this line altogether.